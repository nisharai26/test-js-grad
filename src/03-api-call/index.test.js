const fetchLukeSkywalker = require('./index.js');

describe('fetchLukeSkywalker', () => {
  it('returns correct information about Luke Skywalker', async () => {
    const actualResult = await fetchLukeSkywalker();

    expect(actualResult).toMatchSnapshot();
  });
});
