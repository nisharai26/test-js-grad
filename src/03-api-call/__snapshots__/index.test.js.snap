// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fetchLukeSkywalker returns correct information about Luke Skywalker 1`] = `undefined`;
